app/views/content/life-as-a-teacher/change-careers/benefits-of-changing-careers/_article.html.erb

```diff
@@ -26,7 +26,15 @@
       heading_color: "pink",
     ) %>
   </div>
-  <%= render CallsToAction::AdviserComponent.new(
-    text: "An adviser with years of teaching experience can help you change careers to become a teacher. Chat by phone, text, or email as little or often as you need.",
-  ) %>
+</div>
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::MailinglistBlockComponent.new %>
```

Comment:
I'd suggest switching the order to advisers (most important for this audience), then mailing list, then routes

Reply:
Actually I think it is worth including events in this block?